src/main/resources/db/migration/V3_6__close_oppgaver_made_by_invalid_motesvar.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE person_oppgave
+SET behandlet_tidspunkt = now(),
+    behandlet_veileder_ident = 'X00000',
```

Comment:
En ansattident har vel 7 tegn? Men har kanskje ikke så mye å si

Reply:
Virker lurt å fikse det.

Reply:
Det er feil på forrige script også, burde man endre de som allerede er lagret også?

Reply:
Er det dumt med denne så, er det vel dumt med de andre også? Jeg kan lage en egen pr for det